### ISSUE TITLE
UI Usability

### ISSUE THREAD

**Issue by runleveldev** [mieweb/opensource-server#163]
On mobile and other narrow screen sizes, the content disappears on some pages.

<img width="870" height="996" alt="Image" src="https://github.com/user-attachments/assets/913cc569-6c07-4fbf-9916-9ad31d1ba84c" />

Additionally, the blue-on-blue in the header breadcrumbs is not WCAG compliant contrast.

<img width="342" height="49" alt="Image" src="https://github.com/user-attachments/assets/2a0c6e4e-d769-42d7-8361-992e0f1e6952" />

Both of these issues should be fixed by using reactive Tailwind in place of ALL custom or Bootstrap CSS https://tailwindcss.com/

**Comment by runleveldev** [comment]
Closed by #319
